generators/app/index.js

```diff
@@ -147,9 +157,12 @@ module.exports = class extends Generator {
   writing() {
 
     // Docker Compose environment variables values
-    this.fs.copy(
+    this.fs.copyTpl(
       this.templatePath(this.props.acsVersion + '/.env'),
       this.destinationPath('.env'),
+      {
+        serverName: this.props.serverName
+      }
```

Comment:
I guess `serverName` is not required in `.env` file. Why are you adding this?

Reply:
Forgot to clean that up.